### DISCUSSION TITLE
Understanding the underlying architecture of the PyCaret Time Series Module

### DISCUSSION THREAD

**Discussion by ngupta23** [pycaret/pycaret#1848] [FAQs]
### This article has been migrated to Medium. Please **[check it out here](https://medium.com/towards-data-science/pycaret-time-series-module-architecture-overview-57336a2f39c7)**.

**Comment by moezali1** [comment]
PyCaret 4.0 sunset the legacy time-series module (sktime/prophet integration), so this FAQ no longer maps to a supported code path.

If you still need time-series forecasting, use sktime / prophet / statsforecast directly — those projects have surpassed what the PyCaret wrapper offered.

For 4.0 docs, see https://pycaret.org/docs and the cleanup notice at https://github.com/pycaret/pycaret/issues/4177.

Closing as outdated.
